Cubical/Data/Int/Properties.agda

```diff
@@ -221,6 +333,197 @@ ind-assoc _·_ f g p q base m n (suc o) =
+
+·-pos-suc : ∀ m n → pos (suc m) · n ≡ n + pos m · n
+·-pos-suc m n = signed-distrib (sgn n) (abs n) (m ℕ.· abs n) ∙ λ i → signed-inv n i + signed (sgn n) (m ·ⁿ abs n)
+
+-- this is a similar proof to the one in Cubical.HITs.QuoInt.Properties
+--   but we have less definitional equalities
+-- ·-assoc : ∀ m n o → m · (n · o) ≡ m · n · o
+-- ·-assoc m n o     with abs m | signed-inv m
+-- ... | zero   | pm = {!   !}
+-- ... | suc mⁿ | pm with abs n | signed-inv n
+-- ... | zero   | pn = (λ i → signed s₁ (γ₁ i)) ∙ signed-zero s₁ ∙ sym (signed-zero s₂) ∙ (λ i → signed s₂ (γ₂ i)) where
+--   s₁ = sgn m ⊕ sgn (signed (sgn n ⊕ sgn o) 0)
+--   s₂ = sgn (signed (sgn m ⊕ sgn n) (mⁿ ·ⁿ 0)) ⊕ sgn o
+--   γ₁ = suc mⁿ ·ⁿ abs (signed (sgn n ⊕ sgn o) 0)        ≡⟨ {!   !} ⟩
+--        suc mⁿ ·ⁿ 0                                     ≡⟨ {!   !} ⟩
+--        0                                               ∎
+--   γ₂ = 0                                               ≡⟨ {!   !} ⟩
+--        0 ·ⁿ abs o                                      ≡⟨ {!   !} ⟩
+--        (mⁿ ·ⁿ 0) ·ⁿ abs o                              ≡⟨ {!   !} ⟩
+--        abs (signed (sgn m ⊕ sgn n) (mⁿ ·ⁿ 0)) ·ⁿ abs o ∎
+-- ... | suc nⁿ | pn with abs o | signed-inv o
+-- ... | zero   | po = (λ i → signed s₁ (γ₁ i)) ∙ signed-zero s₁ ∙ sym (signed-zero s₂) ∙ (λ i → signed s₂ (γ₂ i)) where
+--   s₁ = sgn m ⊕ sgn (signed (sgn n ⊕ sgn o) (nⁿ ·ⁿ zero))
+--   s₂ = sgn (signed (sgn m ⊕ sgn n) (suc mⁿ ·ⁿ suc nⁿ)) ⊕ sgn o
+--   γ₁ = suc mⁿ ·ⁿ abs (signed (sgn n ⊕ sgn o) (nⁿ ·ⁿ 0)) ≡[ i ]⟨ suc mⁿ ·ⁿ abs-signed (sgn n ⊕ sgn o) (nⁿ ·ⁿ 0) i ⟩
+--        suc mⁿ ·ⁿ                             (nⁿ ·ⁿ 0)  ≡[ i ]⟨ suc mⁿ ·ⁿ (ℕ.·-nullifiesʳ nⁿ i) ⟩
+--        suc mⁿ ·ⁿ                                    0   ≡⟨ ℕ.·-nullifiesʳ (suc mⁿ) ⟩
+--                                                     0 ∎
+--   γ₂ = 0 ≡⟨ sym $ ℕ.·-nullifiesʳ (abs (signed (sgn m ⊕ sgn n) (suc mⁿ ·ⁿ suc nⁿ))) ⟩
+--        abs (signed (sgn m ⊕ sgn n) (suc mⁿ ·ⁿ suc nⁿ)) ·ⁿ 0 ∎
+-- ... | suc oⁿ | po = cong₂ signed γ₁ γ₂ where
+--   γ₁ = sgn m ⊕ sgn (signed (sgn n ⊕ sgn o) (suc nⁿ ·ⁿ suc oⁿ)) ≡[ i ]⟨ sgn m ⊕ sgn-signed (sgn n ⊕ sgn o) (oⁿ +ⁿ nⁿ ·ⁿ suc oⁿ) i ⟩
+--        sgn m ⊕ (sgn n ⊕ sgn o)                                 ≡⟨ ⊕-assoc (sgn m) (sgn n) (sgn o) ⟩
+--        (sgn m ⊕ sgn n) ⊕ sgn o                                 ≡[ i ]⟨ sgn-signed (sgn m ⊕ sgn n) (nⁿ +ⁿ mⁿ ·ⁿ suc nⁿ) (~ i) ⊕ sgn o ⟩
+--        sgn (signed (sgn m ⊕ sgn n) (suc mⁿ ·ⁿ suc nⁿ)) ⊕ sgn o ∎
+--   γ₂ = suc mⁿ ·ⁿ abs (signed (sgn n ⊕ sgn o) (suc nⁿ ·ⁿ suc oⁿ)) ≡[ i ]⟨ suc mⁿ ·ⁿ abs-signed (sgn n ⊕ sgn o) (suc nⁿ ·ⁿ suc oⁿ) i ⟩
+--        suc mⁿ ·ⁿ (suc nⁿ ·ⁿ suc oⁿ)                              ≡⟨ ℕ.·-assoc (suc mⁿ) (suc nⁿ) (suc oⁿ) ⟩
+--        (suc mⁿ ·ⁿ suc nⁿ) ·ⁿ suc oⁿ                              ≡[ i ]⟨ abs-signed (sgn m ⊕ sgn n) (suc mⁿ ·ⁿ suc nⁿ) (~ i) ·ⁿ suc oⁿ ⟩
+--        abs (signed (sgn m ⊕ sgn n) (suc mⁿ ·ⁿ suc nⁿ)) ·ⁿ suc oⁿ ∎
```

Comment:
I think that there are some definitional equalities missing on `Cubical.Data.Int` to perform the same proof as in  `Cubical.HITs.QuoInt.Properties`. Therefore I adapted the proof from the non-cubical standard library (see below).

TODO: I will delete this comment before merge